utils/_context/_scenarios.py

```diff
@@ -497,11 +498,15 @@ def __init__(self, name, weblog_env) -> None:
         self._required_containers = []
         super().__init__(name, use_proxy=True)
 
+        self._check_env_vars()
```

Comment:
So, you can't call it here, because it would make the presence of those env mandatory everywhere.

Can you call it in `configure()` ? (see below)